tests/OSPSuite.Presentation.Tests/OSPSuite.Presentation.Tests.csproj

```diff
@@ -14,7 +14,7 @@
 
   <ItemGroup>
     <PackageReference Include="FakeItEasy" Version="6.2.1" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.5" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.0" />
```

Comment:
@georgeDaskalakis Downgrading the package?

Reply:
@Yuri05 there were errors because the same package in Core has version 3.1.0. I opted for changing the package version here, since it is just a test project, instead of updating the Core package. Of course if you think it is preferable, I can change update the version in all other projects in the solution to be 3.1.5

Reply:
I think it's ok to keep it like this (would try to make as few changes to Core as possible until V10 release).
Just wanted to make sure it's not an "unintended change" :)

Reply:
Is it then OK if I update the version in Core to 3.1.5? Locally there seem to be no problems with such a change.

Reply:
No. See comment below
The version in core should be 3.1.0. so you're changes are ok (downgrade is good)

Reply:
OK I ll leave it that way. Sorry for the misunderstanding.